sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultServiceSpec.java

```diff
@@ -142,8 +139,8 @@ public String getZookeeperConnection() {
     }
 
     @Override
-    public ReplacementFailurePolicy getReplacementFailurePolicy() {
-        return replacementFailurePolicy;
+    public Optional<ReplacementFailurePolicy> getReplacementFailurePolicy() {
```

Comment:
Let's remove/reword this from the main README `Available and Durable` benefits section too. Down the line we can readdress this.